Tests/Tests/STPRedirectContextTest.m

```diff
@@ -139,18 +139,30 @@ - (void)testInitWithPaymentIntent {
     XCTAssertNil(sut.nativeRedirectURL);
     XCTAssertEqualObjects(sut.redirectURL.absoluteString,
                           @"https://hooks.stripe.com/redirect/authenticate/src_1Cl1AeIl4IdHmuTb1L7x083A?client_secret=src_client_secret_DBNwUe9qHteqJ8qQBwNWiigk");
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
     XCTAssertNotNil(paymentIntent.nextSourceAction.authorizeWithURL.returnURL);
     XCTAssertEqualObjects(sut.returnURL, paymentIntent.nextSourceAction.authorizeWithURL.returnURL);
+#pragma clang diagnostic pop
+    
+    XCTAssertNotNil(paymentIntent.nextAction.redirectToURL.returnURL);
+    XCTAssertEqualObjects(sut.returnURL, paymentIntent.nextAction.redirectToURL.returnURL);
 
     // and make sure the completion calls the completion block above
-    sut.completion(fakeError);
+    XCTAssertNotNil(sut);
+    if (sut.completion) {
+        sut.completion(fakeError);
+    }
     XCTAssertTrue(completionCalled);
 }
 
 - (void)testInitWithPaymentIntentFailures {
+    // Note next_action has been renamed to next_source_action in the API, but both still get sent down in the 2015-10-12 API
     NSMutableDictionary *json = [[STPTestUtils jsonNamed:STPTestJSONPaymentIntent] mutableCopy];
     json[@"next_source_action"] = [json[@"next_source_action"] mutableCopy];
```

Comment:
I don't think you're parsing `next_source_action` anymore, I'd probably clean up this test & remove the mutation of that key.

Reply:
Yep, I didn't convert this test over properly.